superbench/benchmarks/micro_benchmarks/cublas_function.py

```diff
@@ -236,6 +236,12 @@ def add_parser_arguments(self):
             required=False,
             help='The acceptable error bound for correctness check.',
         )
+        self._parser.add_argument(
+            '--random_data',
+            action='store_true',
+            default=False,
+            help='Enable random data generation for performance test.',
```

Comment:
Would you please clarify the default setting in help

Reply:
sure, updated